sdk/guestconfiguration/Microsoft.Azure.Management.GuestConfiguration/tests/ScenarioTests/GuestConfigurationTests.cs

```diff
@@ -20,9 +20,9 @@ public class AutomationTest
         private const string AssignmentName = "AuditSecureProtocol";
```

Comment:
Please revert this change, this file should not be changed.

Reply:
reverted